### DISCUSSION TITLE
Release x64  meet error

### DISCUSSION THREAD

**Discussion by DJDQfff** [stride3d/stride#2333] [Q&A]
It is a template project (latest), when I try to release x64 , It shows the error, it didn't give any useful message.

![image](https://github.com/stride3d/stride/assets/31266256/b5615997-bb22-4cfb-8b46-6ca3820e6c3d)

**Comment by Eideren** [comment]
Check the build output window instead of the error list. It looks like one of you type is not serializable given the third message, does it error out only when building a release or when building a debug one as well ?

**Reply by DJDQfff** [reply]
The solution is the default empty project created by the stride launcher, and the non-serializable content mentioned in the error message is BasicCameraController.cs, which is also the default content.
At first, I can run Debug (x64), but I can't run Release (x64).
Then I don't know what was done, and the project later worked on Releasse (x64).
I tried to delete the obj and bin folders in the Mygame.windows and Mygame projects, and I got the same error again.
After that, I ran Release (Any CPU) and it worked fine, and then I ran Release (x64) and I didn't get any errors
